addOns/grpc/src/main/java/org/zaproxy/addon/grpc/ExtensionGrpc.java

```diff
@@ -22,25 +22,62 @@
 import org.parosproxy.paros.Constant;
 import org.parosproxy.paros.extension.ExtensionAdaptor;
 import org.parosproxy.paros.extension.ExtensionHook;
+import org.zaproxy.zap.extension.httppanel.component.split.request.RequestSplitComponent;
+import org.zaproxy.zap.extension.httppanel.component.split.response.ResponseSplitComponent;
+import org.zaproxy.zap.extension.httppanel.view.HttpPanelView;
+import org.zaproxy.zap.extension.httppanel.view.impl.models.http.request.RequestBodyStringHttpPanelViewModel;
+import org.zaproxy.zap.extension.httppanel.view.impl.models.http.response.ResponseBodyStringHttpPanelViewModel;
+import org.zaproxy.zap.view.HttpPanelManager;
+import org.zaproxy.zap.view.ZapMenuItem;
 
 public class ExtensionGrpc extends ExtensionAdaptor {
 
     public static final String NAME = "ExtensionGrpc";
 
+    private ZapMenuItem protoBufToolsMenuItem;
+
     public ExtensionGrpc() {
         super(NAME);
     }
 
     @Override
     public void hook(ExtensionHook extensionHook) {
         super.hook(extensionHook);
+        if (hasView()) {
+            HttpPanelManager panelManager = HttpPanelManager.getInstance();
+            panelManager.addRequestViewFactory(
+                    RequestSplitComponent.NAME, new RequestGrpcViewFactory());
+            panelManager.addResponseViewFactory(
+                    ResponseSplitComponent.NAME, new ResponseGrpcViewFactory());
+        }
     }
 
     @Override
     public boolean canUnload() {
         return true;
     }
 
+    @Override
+    public void unload() {
+        if (hasView()) {
+            HttpPanelManager panelManager = HttpPanelManager.getInstance();
+            // remove views and their factories
+            panelManager.removeRequestViewFactory(
+                    RequestSplitComponent.NAME, RequestGrpcViewFactory.NAME);
+            panelManager.removeRequestViews(
+                    RequestSplitComponent.NAME,
+                    HttpPanelGrpcView.NAME,
+                    RequestSplitComponent.ViewComponent.BODY);
+            panelManager.removeResponseViewFactory(
+                    ResponseSplitComponent.NAME, ResponseGrpcViewFactory.NAME);
+            panelManager.removeResponseViews(
+                    ResponseSplitComponent.NAME,
+                    HttpPanelGrpcView.NAME,
+                    ResponseSplitComponent.ViewComponent.BODY);
+        }
+        super.unload();
```

Comment:
Not needed.

Reply:
what is not needed only super.unload() or the whole unload part ?

Reply:
Just the `super.unload()`. (If the comment does not have `Comment on lines ...` it means it's only for one.)